go/vt/binlog/binlogplayer/binlog_player_test.go

```diff
@@ -133,25 +145,37 @@ func TestApplyEventsFail(t *testing.T) {
 	}
 }
 
+var settingsFields []*querypb.Field = []*querypb.Field{
+	{Name: "pos", Type: sqltypes.VarBinary},
+	{Name: "stop_pos", Type: sqltypes.VarBinary},
+	{Name: "max_tps", Type: sqltypes.Int64},
+	{Name: "max_replication_lag", Type: sqltypes.Int64},
+	{Name: "state", Type: sqltypes.VarBinary},
+	{Name: "workflow_type", Type: sqltypes.Int64},
+	{Name: "workflow", Type: sqltypes.VarChar},
```

Comment:
This is workflow name, I'm assuming? If so, might be worth being explicit about that (even though it's a reasonable assumption).

Reply:
This again was stolen from #9742 authored by @rohit-nayak-ps. Anyway, `workflow` here is the UUID, not the name. I think it's too late to change this name, it's all over the place.